numpy/__init__.pxd

```diff
@@ -477,16 +470,16 @@ cdef extern from "numpy/arrayobject.h":
     npy_intp PyArray_REFCOUNT(object)
     object PyArray_ContiguousFromAny(op, int, int min_depth, int max_depth)
     unsigned char PyArray_EquivArrTypes(ndarray a1, ndarray a2)
-    bint PyArray_EquivByteorders(int b1, int b2)
+    bint PyArray_EquivByteorders(int b1, int b2) nogil
     object PyArray_SimpleNew(int nd, npy_intp* dims, int typenum)
     object PyArray_SimpleNewFromData(int nd, npy_intp* dims, int typenum, void* data)
     #object PyArray_SimpleNewFromDescr(int nd, npy_intp* dims, dtype descr)
```

Comment:
Why remove this?

Reply:
Or rather, why was it removed previously.

Reply:
OK, I see that there a lot of these commented out without comment. It would be nice to know why, but that is not something for this PR.

Reply:
I can't say. The git history says `Update numpy.pxd` when these were added :)
Feel free to uncomment them, then we should follow in Cython. (Happy to receive a PR :) )

Reply:
Just to expand a bit on the comments above: this file was copied from Cython, where the functions were already commented out (the few I checked were last touched in 2009).